doc/User_Management/User_Management.md

```diff
@@ -0,0 +1,347 @@
+            }
+
+            leaf role {
+                type user-role;
+                mandatory true;
+                description "The role assigned to the user, which determines their group memberships and privileges.";
+            }
+
+            leaf password_hash {
+                type string;
+                mandatory true;
+                must "not(starts-with(., '!'))" {
+                    error-message "Password hash cannot start with '!'. Use the 'enabled' attribute to disable user accounts.";
+                }
+                description "The hashed password string for the user, as found in /etc/shadow. Password hashes can be generated using 'mkpasswd' utility or programmatically using libraries like 'passlib'. To disable an account, use the 'enabled' attribute instead of prepending '!' to the password hash.";
+            }
+
+            leaf-list ssh_keys {
+                type string;
+                description "A list of full public SSH key strings.";
+            }
+
+            leaf enabled {
+                type boolean;
+                default true;
+                description "Whether the user account is enabled. When false, the password is disabled by prepending '!' to prevent password-based login while preserving SSH key access.";
+            }
+        }
+
+        list USER_SECURITY_POLICY_LIST {
+            key "role";
+            description "Global security policies applied to users based on their role.";
+
+            leaf role {
+                type user-role;
+                description "The role for which this security policy applies.";
+            }
+
+            leaf max_login_attempts {
+                type uint32 {
+                    range "1..1000";
+                }
+                description "Maximum number of failed login attempts before accounts with this role are locked. If not set, system defaults apply.";
+            }
+        }
+    }
+}
+```
+
+### 8.3 CLI Enhancements
+The CLI is enhanced with user management and global security policy commands.
+
+**User Add Command:**
+```
+config user add <username> --role <role>
+                          [--password-hash <hash> | --password-prompt]
+                          [--ssh-key <key>]
+                          [--disabled]
+```
+* Multiple `--ssh-key` flags can be provided to build a list.
+* Use `--password-hash` to provide a pre-hashed password directly.
+* Use `--password-prompt` to enter the password securely through an interactive prompt (password will be hashed automatically).
+* Use `--disabled` to create the account in disabled state. Accounts are enabled by default.
+* If neither password option is provided, the user account will be created with password login disabled (password set to `!`).
+
+**User Delete Command:**
+```
+config user del <username>
+```
+
+**User Modify Command:**
+```
+config user modify <username> [--password-hash <hash> | --password-prompt]
```

Comment:
>config user modify [](http://example.com/codeflow?start=0&length=18)

Currently the commands should be "passwd" or "chpasswd". If you invent new command lines, it will be easy to mix use.

It is possible to align with original commands?